amundsen_application/static/js/components/common/ResourceList/PaginatedApiResourceList/index.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import Pagination from 'react-js-pagination';
+import ResourceListItem from 'components/common/ResourceListItem';
+import { Resource } from 'interfaces';
+import * as Constants from '../constants';
+
+import '../styles.scss';
+
+export interface PaginatedApiResourceListProps {
+  activePage: number;
+  emptyText?: string;
+  itemsPerPage: number;
+  onPagination: (pageNumber: number) => void;
+  slicedItems: Resource[];
+  totalItemsCount: number;
+  source: string;
+}
+
+class PaginatedApiResourceList extends React.Component<PaginatedApiResourceListProps, {}> {
+  public static defaultProps: Partial<PaginatedApiResourceListProps> = {
+    emptyText: Constants.DEFAULT_EMPTY_TEXT,
+  };
+
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Probably not necessary, maybe something left from of a previous version?

Reply:
You're right not necessary. This isn't unique to this file though, "copy-paste" has led to at least a few more `constructor(props) { super(props) }` in a few other files I'll make a note to remove them on a future code-cleanup-only PR.

Reply:
+1